src/lib/mina_base/account.ml

```diff
@@ -946,7 +966,7 @@ let gen_timing_at_least_one_vesting_period (account_balance : Balance.t) =
   let%bind initial_minimum_balance = Balance.(gen_incl one account_balance)
   (* vesting period must be at least one to avoid division by zero *)
   and cliff_time, vesting_end, vesting_period =
-    gen_with_vesting_period @@ Global_slot.of_int 2
+    gen_with_vesting_period @@ Global_slot_span.of_int 2
```

Comment:
I've seen a lot of `of_int 2` until then. Worth having a `two` constant in the module?